src/eval-frame/components/reps/__test_helpers__/reps-test-value-cases.js

```diff
@@ -318,24 +318,39 @@ export const promiseCases = {
 // ==================== tabular
 
 export const rowTableCases = {
-  rowsTable_plainObjects: new Array(100).fill(0).map((x, i) => ({
+  rowsTable_plainObjects: new Array(58).fill(0).map((x, i) => ({
     index: i,
     time: new Date(),
     id: Math.sin(i)
   })),
-  rowsTable_compositeObjects: new Array(100).fill(compositeObjects),
-  rowsTable_objectsContainingSimpleTypes: new Array(100).fill(simpleTypes),
+
```

Comment:
added a couple cases of tables that don't just repeat the same rows over and over